libs/sdk-ui-ext/src/internal/translations/en-US.json

```diff
@@ -706,6 +706,16 @@
         "comment": "applicable for funnel chart special data labels which can include percentages",
         "limit": 0
     },
+    "properties.canvas.continuousLine.label": {
+        "value": "Continuous line",
+        "comment": "",
+        "limit": 0
+    },
+    "properties.canvas.continuousLine.tooltip": {
+        "value": "Draw a dashed line between the points with missing values.",
```

Comment:
The tooltip should be "Draw a solid line between points with missing values"

Reply:
I have fixed this.  Thanks!